config/routes.rb

```diff
@@ -17,6 +17,7 @@
 
   get "check" => "application#check"
 
+  resource :cookies, controller: :cookies, only: %i[show update]
```

Comment:
It does no harm, nonetheless controller: :cookies is not needed here as we are just using the default.

Reply:
good to know :)